### PR TITLE
Documentation Azure Load Balancer

### PR DESCRIPTION
**Pull request by sachin-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 [Azure Load Balancer](https://learn.microsoft.com/en-us/azure/load-balancer/load-balancer-overview) is an Azure service that allows you to evenly distribute incoming network traffic across a group of Azure VMs or instances in a Virtual Machine Scale Set. This integration helps in monitoring inbound and outbound data throughput, outbound flows, and application endpoint's health  of your Load Balancers.
 
-## Metric types
+## Log and metric types
+For Azure Load Balancer, you can collect the following logs:
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:14]: For Azure Load Balancer, you can collect the following logs and metrics:
```suggestion
For Azure Load Balancer, you can collect the following logs:
```

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+* **Load Balancer Health Event.** No healthy backends log describes your load balancer’s frontend has no healthy backend instances to distribute traffic to.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:15]:
```suggestion
* **Load Balancer Health Event.** These health event logs are emitted when any issues affecting your load balancer’s health and availability are detected. [Learn more](https://techcommunity.microsoft.com/t5/azure-networking-blog/introducing-azure-load-balancer-health-event-logs/ba-p/4154362).```

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 3]
```diff
@@ -20,24 +22,177 @@ For more information on supported metrics, refer to [Azure documentation](https:
 
 ## Setup
 
+:::note
+This app supports only load balancers of type **Standard** and **Gateway**.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:26]:
```suggestion
This app supports only load balancers of **Standard** and **Gateway** SKU.
```

Comment (himanshu219) [reply docs/integrations/microsoft-azure/azure-load-balancer.md:26]: Type is categorized as  Public and Internal hence using the SKU keyword 
image

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 4]
```diff
@@ -20,24 +22,177 @@ For more information on supported metrics, refer to [Azure documentation](https:
...
+### Administrative Operations
+
+The **Azure Load Balancer - Administrative Operations** dashboard provides details like distribution by operation type, by operation, recent delete operations, top 10 operations that caused most errors and users / applications by operation type etc.
+
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureLoadBalancer/Azure-Load-Balancer-Administrative-Operations.png')} alt="Azure Load Balancer Administrative Operations" style={{border: '1px solid gray'}} width="800" />
+
+
+### Health
+
+The **Azure Load Balancer - Health** dashboard provides details like total failed requests, failures by operation, health probe status trend and unhealthy backends etc.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:178]: Use this dashboard to

1. Identify failed requests and operations
2. Detect when all backend instances in a pool are not responding to the configured health probes.

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 5]
```diff
@@ -20,24 +22,177 @@ For more information on supported metrics, refer to [Azure documentation](https:
...
+### Overview
+
+The **Azure Load Balancer - Overview** dashboard provides details like Health Probe Status, Average Data Path Availability, Transmission Details, Connection Details and SNAT Ports Utilization(%) etc.
+
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureLoadBalancer/Azure-Load-Balancer-Overview.png')} alt="Azure Load Balancer Overview" style={{border: '1px solid gray'}} width="800" />
+
+
+### Administrative Operations
+
+The **Azure Load Balancer - Administrative Operations** dashboard provides details like distribution by operation type, by operation, recent delete operations, top 10 operations that caused most errors and users / applications by operation type etc.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:171]: Use this dashboard to:

Identify top users performing administrative operations.
View Top 10 operations that caused the most errors.
View recent read, write, and delete operations.

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by himanshu219 — APPROVED** [review]

File: docs/integrations/microsoft-azure/azure-load-balancer.md [hunk 6]
```diff
@@ -10,34 +10,207 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+- Detect when there is less data path availability than expected due to platform issues.
+- Monitor data transmission (packets and bytes) through your load balancers.
+
+<img src={useBaseUrl('https://sumologic-app-data.s3.amazonaws.com/dashboards/AzureLoadBalancer/Azure-Load-Balancer-Network.png')} alt="Azure Load Balancer Network" style={{border: '1px solid gray'}} width="800" />
+
+### Policy
+
+The **Azure Load Balancer - Policy** dashboard provides details like total success policy events, success policy events, total failed policy events, and failed policy events.
+
+Use this dashbaord to:
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-load-balancer.md:209]:
```suggestion
Use this dashboard to: 
```

**Pull request merged**